### ISSUE TITLE
Low expected f1score

### ISSUE THREAD

**Issue by richardzhangy26** [VinAIResearch/MISCA#1]
when I train the MISCA model, the following command is  
`python main.py --token_level word-level \
            --model_type roberta \
            --model_dir misca \
            --task mixatis \
            --data_dir data \
            --attention_mode label \
            --do_train \
            --do_eval \
            --num_train_epochs 100 \
            --intent_loss_coef 0.5 \
            --learning_rate 1e-5 \
            --num_intent_detection \
            --use_crf \ 
            --base_model dir_base \
            --intent_slot_attn_type coattention`
Finally, I got a low expected f1 score. What's wrong with it?
![image](https://github.com/VinAIResearch/MISCA/assets/77674465/f43b1d89-df4f-4d68-bfd4-3639feabbd00)

**Comment by thinhphp** [comment]
Thanks for your interest in our work! Our model first trains the "base" model without coattention mechanism first, and loads this pretrained base model to learn with coattention afterward. Hope it is helpful for you.

**Comment by dengg1013** [comment]
I first train the base model using bert backbone,the following command is
python main.py --token_level word-level \
            --model_type bert \
            --model_dir dir_base \
            --task my dataset \
            --data_dir data \
            --attention_mode label \
            --do_train \
            --do_eval \
            --num_intent_detection \
            --use_crf,
and then loads dir_base model,the  following command is
python main.py --token_level word-level \
            --model_type bert \
            --model_dir misca \
            --task my dataset \
            --data_dir data \
            --attention_mode label \
            --do_train \
            --do_eval \
            --num_intent_detection \
            --use_crf \ 
            --base_model dir_base \
            --intent_slot_attn_type coattention,
however, the result still low.
![image](https://github.com/VinAIResearch/MISCA/assets/121504378/ee4b9f5a-3b31-4ce1-b23f-81e6b154ffa7)
